### ISSUE TITLE
TypeError: paraphrase_mining() takes from 2 to 4 positional arguments but 8 were given

### ISSUE THREAD

**Issue by dk-wei** [UKPLab/sentence-transformers#825]
Hi, I met this `TypeError` issue when I'm running the quora training example file today (2020/03/20): `application_duplicate_questions_mining.py`.
This example was running well before.

File Path:
`sentence-transformers/examples/training/quora_duplicate_questions/application_duplicate_questions_mining.py/`

Error Info:

```
2021-03-20 18:55:52 - Evaluate model without training
2021-03-20 18:55:52 - Binary Accuracy Evaluation of the model on  dataset in epoch 0 after 0 steps:
2021-03-20 18:56:36 - Accuracy with Cosine-Similarity:           76.61	(Threshold: 0.8349)
2021-03-20 18:56:36 - F1 with Cosine-Similarity:                 69.29	(Threshold: 0.7391)
2021-03-20 18:56:36 - Precision with Cosine-Similarity:          60.18
2021-03-20 18:56:36 - Recall with Cosine-Similarity:             81.64
2021-03-20 18:56:36 - Average Precision with Cosine-Similarity:  71.80

2021-03-20 18:56:36 - Accuracy with Manhatten-Distance:           76.70	(Threshold: 193.7357)
2021-03-20 18:56:36 - F1 with Manhatten-Distance:                 69.44	(Threshold: 240.4903)
2021-03-20 18:56:36 - Precision with Manhatten-Distance:          59.75
2021-03-20 18:56:36 - Recall with Manhatten-Distance:             82.88
2021-03-20 18:56:36 - Average Precision with Manhatten-Distance:  71.88

2021-03-20 18:56:37 - Accuracy with Euclidean-Distance:           76.75	(Threshold: 8.8834)
2021-03-20 18:56:37 - F1 with Euclidean-Distance:                 69.45	(Threshold: 10.8947)
2021-03-20 18:56:37 - Precision with Euclidean-Distance:          59.90
2021-03-20 18:56:37 - Recall with Euclidean-Distance:             82.62
2021-03-20 18:56:37 - Average Precision with Euclidean-Distance:  71.91

2021-03-20 18:56:37 - Paraphrase Mining Evaluation on dev dataset in epoch 0 after 0 steps:
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-16-8b1c2faf7577> in <module>()
      5 
      6 logger.info("Evaluate model without training")
----> 7 seq_evaluator(model, epoch=0, steps=0, output_path=model_save_path)
      8 
      9 logger.info("Evaluate model with training")

1 frames
/usr/local/lib/python3.7/dist-packages/sentence_transformers/evaluation/SequentialEvaluator.py in __call__(self, model, output_path, epoch, steps)
     16         scores = []
     17         for evaluator in self.evaluators:
---> 18             scores.append(evaluator(model, output_path, epoch, steps))
     19 
     20         return self.main_score_function(scores)

/usr/local/lib/python3.7/dist-packages/sentence_transformers/evaluation/ParaphraseMiningEvaluator.py in __call__(self, model, output_path, epoch, steps)
     87 
     88         #Compute embedding for the sentences
---> 89         pairs_list = paraphrase_mining(model, self.sentences, self.show_progress_bar, self.batch_size,  self.query_chunk_size,  self.corpus_chunk_size, self.max_pairs, self.top_k )
     90 
     91 

TypeError: paraphrase_mining() takes from 2 to 4 positional arguments but 8 were given
````

**Comment by nreimers** [comment]
Hi,
the latest release has splitted the paraphrase_mining into two function:
https://github.com/UKPLab/sentence-transformers/blob/master/sentence_transformers/util.py#L72

What you need to do is to pass everything after self.batch_size with a named parameter, i.e.
pairs_list = paraphrase_mining(model, self.sentences, self.show_progress_bar, self.batch_size,  query_chunk_size=self.query_chunk_size,  corpus_chunk_size=self.corpus_chunk_size, max_pairs=self.max_pairs, top_k=self.top_k )

Will update the code soon and fix it.

Thanks for reporting

**Comment by nreimers** [comment]
It is fixed in the version 1.0.3
